lib/smart_answer/calculators/landlord_immigration_check_calculator.rb

```diff
@@ -19,7 +18,7 @@ module LandlordImmigrationCheck
     )
 
     def self.valid_postcode(postcode)
```

Comment:
You've been adding a lot of question marks in other commits, can I request a bonus feature to add a `?` to the end of this method as it always returns a boolean value? :)